lib/afc/index.js

```diff
@@ -245,5 +262,24 @@ class AfcService {
   }
 }
 
+class FileInfo {
+  constructor ({st_size, st_blocks, st_nlink, st_ifmt, st_mtime, st_birthtime}) {
```

Comment:
these st_ prefixes don't comply with the general js code style

Reply:
I know but it is what the phone sends

Reply:
Seems like since this is entirely internal the value of translating back and forth is low.

Reply:
I can make change the class fields to what `fs.stat()` returns. Is everybody cool with that?

Reply:
Sure!